### PR TITLE
RavenDB-13655 skipping test that throws SO on 32-bit and avoiding AVE…

### PR DESCRIPTION
**Pull request by talweiss1982**
… on cluster tx one by one

### CODE REVIEW

File: test/SlowTests/Issues/RavenDB_10621.cs [hunk 1]
```diff
@@ -9,15 +9,14 @@ namespace SlowTests.Issues
 {
     public class RavenDB_10621 : RavenTestBase
     {
-        [Fact]
+        [Fact(Skip = "Waiting for RavenDB-13830 to be resolved")]
```
Comment (ayende) [inline test/SlowTests/Issues/RavenDB_10621.cs:12]: How does this generate this error?

Comment (talweiss1982) [reply test/SlowTests/Issues/RavenDB_10621.cs:12]: It doesn't it's an old test Michael wrote now it throws SO

Comment (maximburyak) [reply test/SlowTests/Issues/RavenDB_10621.cs:12]: @ayende I think that this issue is related to an old issue that we had: https://github.com/dotnet/coreclr/issues/14672

Comment (arekpalinski) [reply test/SlowTests/Issues/RavenDB_10621.cs:12]: Yes, we wrote this test to make sure that we're correctly handling `InvalidProgramException` during indexing. Once https://github.com/dotnet/coreclr/issues/14672 got fixed we adjusted the test. See the following commit:

https://github.com/ravendb/ravendb/commit/de998b8b9d36b181dfc95f69b4733c3074a4bb23#diff-452a08ab6d945d7e51d8683da35507b3

Note that Pawel changed `[Fact64Bit]` to `[Fact]`. That's why we never saw any failures on 32bits before probably.

Investigation is reported in: https://issues.hibernatingrhinos.com/issue/RavenDB-13830

**Comment by arekpalinski** [comment]
@talweiss1982 can we close this PR. the same changes we have in https://github.com/ravendb/ravendb/pull/9511 right?

**Comment by talweiss1982** [comment]
Yes

**Comment by arekpalinski** [comment]
Closing in favor of https://github.com/ravendb/ravendb/pull/9511

**Pull request closed without merging**